docs/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -490,7 +490,7 @@ Learn more about fixing these caching issues in the <PlatformLink to="/sourcemap
 
   </Expandable>
 
-  <Expandable permalink title="pnpm: Resolving 'import-in-the-middle' external package errors">
+  <Expandable title="pnpm dev: Cannot find package 'import-in-the-middle'">
```

Comment:
q: Is this only an issue for `pnpm dev`? I thought we don't support dev mode anyway?

Reply:
Yes, only in dev mode. Just tested it and building and running the build output works with pnpm

Reply:
Hm ok, I wonder if we then should even bother with installing iitm in the wizard flow (https://github.com/getsentry/sentry-wizard/pull/727)

Reply:
I think it's fine to install it, people will probably want to run Sentry in the dev mode as well. What about changing the prompt to something like this "To run Sentry in pnpm dev, 'import-in-the-middle' needs to be installed."

Reply:
+1 to devs wanting to run Sentry in dev mode.